staging/src/k8s.io/cloud-provider/controllers/route/route_controller_test.go

```diff
@@ -32,6 +32,8 @@ import (
 	fakecloud "k8s.io/cloud-provider/fake"
 	nodeutil "k8s.io/component-helpers/node/util"
 	netutils "k8s.io/utils/net"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Do we really need additional dep? Was that dep already there (used by something else)?

Reply:
It has been used here: https://github.com/kubernetes/kubernetes/blame/master/staging/src/k8s.io/cloud-provider/controllers/node/node_controller_test.go#L40